OSSSpeechKit/Classes/OSSSpeech.swift

```diff
@@ -535,6 +544,40 @@ public class OSSSpeech: NSObject {
             delegate?.didFailToCommenceSpeechRecording()
             delegate?.didFailToProcessRequest(withError: OSSSpeechKitErrorType.invalidSpeechRequest.error)
         }
+        
+        readyToRecord()
+    }
+    
+    func readyToRecord()
```

Comment:
Should this be a private function? Or will apps that use this pod be able to access this? 

Could you please add code doc comments (`///`) about what it does?